pkg/networkservice/core/next/tests/client_test.go

```diff
@@ -31,6 +31,9 @@ import (
 func TestNewNetworkServiceClientShouldNotPanic(t *testing.T) {
 	assert.NotPanics(t, func() {
 		_, _ = next.NewNetworkServiceClient().Request(context.Context(nil), nil)
+		_, _ = next.NewWrappedNetworkServiceClient(func(client networkservice.NetworkServiceClient) networkservice.NetworkServiceClient {
```

Comment:
Probably it would be better to create another `TestNewWrappedNetworkServiceClientShouldNotPanic` test scenario instead of replacing the existing one.

Reply:
Hmm, actually I guess this line is not needed because NewWrappedNetworkServiceClient uses inside NewNetworkServiceClient :)

Reply:
It is not replacing just expanding.